x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/transforms/TransformTaskParams.java

```diff
@@ -50,28 +54,55 @@ public class TransformTaskParams implements SimpleDiffable<TransformTaskParams>,
         PARSER.declareLong(ConstructingObjectParser.optionalConstructorArg(), FROM);
         PARSER.declareString(ConstructingObjectParser.optionalConstructorArg(), FREQUENCY);
         PARSER.declareBoolean(ConstructingObjectParser.optionalConstructorArg(), REQUIRES_REMOTE);
+        PARSER.declareBoolean(ConstructingObjectParser.optionalConstructorArg(), UNATTENDED);
     }
 
-    private TransformTaskParams(String transformId, String version, Long from, String frequency, Boolean remote) {
+    private TransformTaskParams(String transformId, String version, Long from, String frequency, Boolean remote, Boolean unattended) {
         this(
             transformId,
             version == null ? null : TransformConfigVersion.fromString(version),
             from == null ? null : Instant.ofEpochMilli(from),
             frequency == null ? null : TimeValue.parseTimeValue(frequency, FREQUENCY.getPreferredName()),
-            remote == null ? false : remote.booleanValue()
+            remote != null && remote,
+            unattended
         );
     }
 
-    public TransformTaskParams(String transformId, TransformConfigVersion version, TimeValue frequency, boolean remote) {
-        this(transformId, version, null, frequency, remote);
-    }
-
-    public TransformTaskParams(String transformId, TransformConfigVersion version, Instant from, TimeValue frequency, boolean remote) {
+    public TransformTaskParams(
```

Comment:
Do we need to add `unattended` parameter to this constructor?
It was left as a shortcut when we introduced `from` and I think we can leave it like this.